akka-sse/src/main/scala/de/heikoseeberger/akkasse/ServerSentEvent.scala

```diff
@@ -21,6 +21,12 @@ import scala.annotation.tailrec
 
 object ServerSentEvent {
 
+  sealed trait EventIdValue
+  case object ResetId extends EventIdValue
+  final case class Id(id: String) extends EventIdValue {
```

Comment:
After some reconsideration I don't think we need `EventIdValue` at all. Why not go with a plain `Option[String]` like `ServerSentEvent.eventType`?